bin/gamedig.js

```diff
@@ -65,4 +67,10 @@ gamedig.query(options)
 
             printOnPretty({error: error});
         }
+    })
+    .then(() => {
+      // https://github.com/denoland/deno/issues/20138
+      if (typeof Deno !== "undefined") {
+        gamedig.queryRunner.udpSocket?.socket?.close();
+      }
```

Comment:
I dont exactly understand something, `unref` is not implemented, but why do we close the connection only here on the binary one? wouldn't we need to also close instances after being used in lib-only?

Reply:
Yes, sorry I should've mentioned that. I just wasn't sure how to do that on the library as there's no real close method. I guess maybe could close after every query and reconstruct it for the next.